locales/en.json

```diff
@@ -1,15 +1,15 @@
 {
 	"Collaborative markdown notes": "Collaborative markdown notes",
-	"Realtime collaborative markdown notes on all platforms.": "Realtime collaborative markdown notes on all platforms.",
-	"Best way to write and share your knowledge in markdown.": "Best way to write and share your knowledge in markdown.",
+	"Realtime collaborative markdown notes on all platforms.": "Collaborate on markdown notes on all platforms in realtime.",
+	"Best way to write and share your knowledge in markdown.": "<i>the simultaneous markdown editor</i>",
```

Comment:
Please keep the abstraction. Don't mix up translations with HTML.

And I'm not sure if that's the best change :/
I would go for `The best way to write and share your knowledge in markdown.`

Reply:
Can you send a PR to my PR's branch with your changes?

Reply:
Has landed…